internal/sapsystem/sapsystem.go

```diff
@@ -255,6 +258,15 @@ func landscapeHostConfiguration(sid, instance string) map[string]interface{} {
 	return runPythonSupport(sid, instance, "landscapeHostConfiguration.py")
 }
 
+func hdbnsutilSrstate(sid, instance string) map[string]interface{} {
+	user := fmt.Sprintf("%sadm", strings.ToLower(sid))
+	cmdPath := path.Join("/usr/sap", sid, instance, "exe", "hdbnsutil")
```

Comment:
We should probably use `sapInstallationPath` there instead of `"/usr/sap"` right?